### PR TITLE
Update YAML instance form to be used for edit only [WD-1358]

### PR DESCRIPTION
**Pull request by lorumic**

### CODE REVIEW

File: static/js/pages/instances/EditInstanceForm.tsx [hunk 1]
```diff
@@ -65,48 +67,22 @@ const InstanceFormYaml: FC = () => {
...
-            </h4>
+            <h4>Edit instance configuration for {panelParams.instance}</h4>
           }
         />
         <NotificationRow notify={notify} />
-        {isEditMode && isLoading ? (
+        {isLoading ? (
           <Loader text="Loading instance details..." />
-        ) : (
+        ) : instance ? (
```
[the remark this answers is not in the document]
Comment (lorumic) [reply static/js/pages/instances/EditInstanceForm.tsx:81]: So I should change it to something like:
```
{isLoading && (loader)}
{!isLoading && instance ? (show) : (error)}
```
Is this what you mean? Or something different?

[the remark this answers is not in the document]
Comment (edlerd) [reply static/js/pages/instances/EditInstanceForm.tsx:81]: Either that or nesting might be easier to digest: `isLoading ? (load) : ( instance ? (show) : (error) )`. I have no strong opinion which is cleaner.

[the remark this answers is not in the document]
Comment (lorumic) [reply static/js/pages/instances/EditInstanceForm.tsx:81]: > Either that or nesting might be easier to digest: `isLoading ? (load) : ( instance ? (show) : (error) )`. I have no strong opinion which is cleaner.

That makes the linter unhappy:

```
   81:13  error  Delete `(`  prettier/prettier
  108:9   error  Delete `)`  prettier/prettier

✖ 2 problems (2 errors, 0 warnings)
  2 errors and 0 warnings potentially fixable with the `--fix` option.
```

**Comment by lorumic** [comment]
Both points addressed @ [e82c553](https://github.com/canonical/lxd-ui/commit/e82c553fa5393e5413368f3649fae4338a182fdf)

File: static/js/pages/instances/EditInstanceForm.tsx [hunk 2]
```diff
@@ -121,20 +96,18 @@ const InstanceFormYaml: FC = () => {
...
-                        : "Create instance"
-                    }
+                    buttonLabel="Update instance"
                   />
                 </Col>
               </Row>
             </Form>
           </Row>
+        ) : (
+          <>Could not load instance details.</>
```
[the remark this answers is not in the document]
Comment (lorumic) [reply static/js/pages/instances/EditInstanceForm.tsx]: Done @ [1ff323c](https://github.com/canonical/lxd-ui/commit/1ff323c2faf24a351353cf4302ed59d977eedff2)

**Review by edlerd — APPROVED** [review]

**Pull request merged**